### ISSUE TITLE
Minor Annoyance: communication toggles are inverted

### ISSUE THREAD

**Issue by Darin755** [jeremyckahn/chitchatter#157]
## The problem

If you visit chitchatter right now you'll notice that the media toggles are backward. They are green when inactive but red and crossed when active. This is very confusing and can lead to puzzled users

## A solution

What should happen is that they should be crossed out and greyed out while they are disabled and permissions are missing. When they are clicked there should be a check to see if the user granted the proper permissions and if they did it should just be a clear icon with minimal colors. If something goes wrong they should turn red with a X over the top.

## example

[Jitsi](jitsi.vern.cc) is a good example of this type of animation

**Comment by jeremyckahn** [comment]
@Darin755 I just merged and deployed #164. I decided to deviate from my original plan of using Google meet as a reference design. I did this because Google Meet's design emphasizes video and audio chat, whereas Chitchatter emphasizes text chat. Given this, having red as a default (disabled) button color seemed distracting, so I opted for green representing "enabled" and grey representing "disabled."

Let me know what you think!
